editor/src/app/profile-dropdown/profile-dropdown.component.ts

```diff
@@ -44,10 +55,21 @@ import { Router } from '@angular/router';
     }
   `]
 })
-export class ProfileDropdownComponent {
+export class ProfileDropdownComponent implements OnInit {
+
+  user: Observable<UserStateModel>;
```

Comment:
This type is incorrect. You say here, that it's an observable, but actually, assign `UserStateModel` to it in line `71`.
You can either set it to `UserStateModel` type here (`user: UserStateModel`) or assign the observable to this variable and use `async` pipe in the template (`@Select(UserState) user: Observable<UserStateModel>`, `{{ (user | async).name }}`)